TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/NoteDetailFragment.kt

```diff
@@ -380,6 +409,19 @@ class NoteDetailFragment : Fragment() {
         }
     }
 
+    private fun openShareIntent(bitmap: Bitmap, path: String) {
+        val outputStream = FileOutputStream(path)
+        bitmap.compress(Bitmap.CompressFormat.JPEG, 100, outputStream)
```

Comment:
what is the `100` here? add a constant so user knows what value they are modifying